sdk/spring/spring-cloud-azure-starter-monitor/README.md

```diff
@@ -78,20 +78,24 @@ dependencyManagement {
 }
 ```
 
-### Authentication
+### Azure Monitor configuration
 
-#### Get the connection string from the portal
+#### Authentication
```

Comment:
I think Authentication in the context of Azure Monitor normally means AAD, not Connection String

Reply:
Please check 0b96ccdc7605ac5f58285ed36b48fea0d95d73d5